lib/std/fmt.zig

```diff
@@ -1885,7 +1885,7 @@ pub fn bufPrintZ(buf: []u8, comptime fmt: []const u8, args: anytype) BufPrintErr
 /// Count the characters needed for format. Useful for preallocating memory
 pub fn count(comptime fmt: []const u8, args: anytype) u64 {
     var counting_writer = std.io.countingWriter(std.io.null_writer);
-    format(counting_writer.writer(), fmt, args) catch |err| switch (err) {};
+    format(counting_writer.writer(), fmt, args) catch unreachable;
```

Comment:
What's going on here? Ideally we would keep the previous code so that if a new error is introduced to the empty error, set we get a compile error here instead of asserting no error happens.

Reply:
This is #11306, I'll add a TODO comment.

Reply:
c21f046a8b019c42aa0dbb0fb9c592b590edf977 :grin:

Reply:
Guess I'll just close this pr at this point and implement input variables in assembly.